tests/e2e/factories/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@woocommerce/e2e-factories",
+  "version": "0.1.0",
+  "author": "Automattic",
+  "description": "Factories for generating test data for use in WooCommerce End-To-End tests.",
+  "homepage": "https://github.com/woocommerce/woocommerce/tree/master/tests/e2e/factories/README.md",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/woocommerce/woocommerce.git"
+  },
+  "keywords": [
+    "woocommerce",
+    "e2e"
+  ],
+  "license": "GPL-3.0+",
+  "main": "dist/index.js",
+  "sideEffects": false,
+  "scripts": {
+    "build": "tsc",
+    "test": "jest"
+  },
+  "dependencies": {
+    "@wordpress/api-fetch": "^3.14.0",
+    "create-hmac": "1.1.7",
+    "fishery": "file:./lib/fishery/",
```

Comment:
I share Darren concern, why are we including this manually?

Reply:
We're currently blocked on including via NPM by [this issue](https://github.com/thoughtbot/fishery/issues/16). The version that we're including here comes from the `async` branch, as I didn't want to be stalled over the weekend in working on this.